### ISSUE TITLE
Area of a circle is pi*r^2

### ISSUE THREAD

**Issue by anzelpwj** [AtomicScala/atomic-scala-examples#10]
I realize the exercise is more with the practice of coding than any actual computational geometry, but thought I might as well point this out. In `Shape_Inheritance.scala` and `Shape_TypeClass.scala`, you're using the circumference formula for a circle, not the area formula (`Pi * radius * radius`).

Just a heads up.

**Comment by BruceEckel** [comment]
No, cake are square! Pie are round. (An engineering joke told by an engineer in my first "real" job out of college).

Thanks for pointing it out. I've been immersed in Atomic Kotlin so it might be awhile before I get to this.

**Comment by anzelpwj** [comment]
As a "devout" [tau-ist](https://tauday.com/) I'd say ideally we'd use the true circle constant. ;)

No worries on addressing this sooner rather than later. Enjoyed the book!
